python/auto-instrumentation/README.md

```diff
@@ -0,0 +1,81 @@
+#Steps to auto-instrument Python app
+###Step1: Get your python app
+You can replace the server.py app with your python app.
+
+###Step2: Installing OpenTelemetry Components
+The 'requirements.txt' file contains all the necessary commands to set up OpenTelemetry Python instrumentation. 
+All the mandatory packages required to start the instrumentation are installed with the help of this file. 
+```
+pip3 install -r requirements.txt
+```
+###Step3: Installing application specific packages
+Run the following command from our application directory to install all instrumented packages used in our application.
+```
+opentelemetry-bootstrap --action=install
+```
+
+###Step4: Installing and Configuring the OpenTelemetry Exporter
+Run following command to install the OpenTelemetry exporter and configure it to send traces from our application to the required endpoint on our local machine.
+```
+pip3 install opentelemetry-exporter-otlp
+```
+Configure environment variables specific to our exporter.
```

Comment:
I think it would be nice to add a note that the service name should be customized, unless there's a reason we want folks to use `myServiceName`

Reply:
Done!